app/addons/replication/route.js

```diff
@@ -10,48 +10,67 @@
 // License for the specific language governing permissions and limitations under
 // the License.
 
-import app from "../../app";
-import FauxtonAPI from "../../core/api";
-import Replication from "./resources";
-import Views from "./views";
-var RepRouteObject = FauxtonAPI.RouteObject.extend({
-  layout: 'one_pane',
+import FauxtonAPI from '../../core/api';
+import ReplicationController from './controller';
+import ComponentActions from '../components/actions';
+
+const ReplicationRouteObject = FauxtonAPI.RouteObject.extend({
+  layout: 'empty',
+  hideNotificationCenter: true,
+  hideApiBar: true,
+
   routes: {
-    "replication": 'defaultView',
-    "replication/:dbname": 'defaultView'
+    'replication/create': 'defaultView',
+    'replication/:dbname': 'defaultView',
```

Comment:
it looks like you can't edit a database with the name `create` using this routing